utils/DeploymentUtil.js

```diff
@@ -31,62 +31,88 @@ const deployDao = async (options) => {
   const {
     deployFunction,
     owner,
-    creator,
     deployTestTokens,
     finalize,
     DaoRegistry,
     BankExtension,
     BankFactory,
     NFTExtension,
     NFTCollectionFactory,
+    ERC20TokenExtension,
+    ERC20TokenExtensionFactory,
     TestToken1,
     TestToken2,
     Multicall,
     PixelNFT,
     OLToken,
   } = options;
-
   let identityDao = await deployFunction(DaoRegistry);
+
   let identityBank = await deployFunction(BankExtension);
   let bankFactory = await deployFunction(BankFactory, [identityBank.address]);
+
   let identityNft = await deployFunction(NFTExtension);
   let nftFactory = await deployFunction(NFTCollectionFactory, [
     identityNft.address,
   ]);
 
+  let identityERC20Ext = await deployFunction(ERC20TokenExtension);
+  let erc20TokenExtFactory = await deployFunction(ERC20TokenExtensionFactory, [
+    identityERC20Ext.address,
+  ]);
+
   const { dao, daoFactory } = await cloneDao({
     ...options,
     identityDao,
     name: options.daoName || "test-dao",
   });
 
+  // Start the BankExtension deployment and configuration
   await bankFactory.createBank(options.maxExternalTokens);
   let pastEvent;
   while (pastEvent === undefined) {
     let pastEvents = await bankFactory.getPastEvents();
     pastEvent = pastEvents[0];
   }
-
   const { bankAddress } = pastEvent.returnValues;
   const bankExtension = await BankExtension.at(bankAddress);
   await dao.addExtension(sha3("bank"), bankExtension.address, owner, {
     from: owner,
   });
 
+  // Start the NFTExtension deployment and configuration
   await nftFactory.createNFTCollection();
   pastEvent = undefined;
   while (pastEvent === undefined) {
     let pastEvents = await nftFactory.getPastEvents();
     pastEvent = pastEvents[0];
   }
-
   const { nftCollAddress } = pastEvent.returnValues;
   const nftExtension = await NFTExtension.at(nftCollAddress);
   await dao.addExtension(sha3("nft"), nftCollAddress, owner, {
     from: owner,
   });
 
-  const extensions = { bank: bankExtension, nft: nftExtension };
+  // Start the Erc20TokenExtension deployment & configuration
+  await erc20TokenExtFactory.create("Unit Token", UNITS, "UNIT", 18);
```

Comment:
As discussed with @mcchan1, we should have the ability to override the token name and symbol here with custom values passed in from `2_deploy_contracts.js` and env vars.

Reply:
@jdville03 we just updated that to use env vars.